<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Merchant</class>
 <widget class="QMainWindow" name="Merchant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>账户交易</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="HAll">
      <item>
       <layout class="QVBoxLayout" name="Vbt">
        <item>
         <widget class="QPushButton" name="btSaveMoney">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>存钱</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btGetMoney">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>取钱</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btTransMoney">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>转账</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btAccountManage">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>账户管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btLogOut">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>退出登录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>OperateLog</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextBrowser" name="ShowLog"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,3">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btShowLog">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>查询日志</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>btSaveMoney</tabstop>
  <tabstop>btGetMoney</tabstop>
  <tabstop>btTransMoney</tabstop>
  <tabstop>btAccountManage</tabstop>
  <tabstop>btLogOut</tabstop>
  <tabstop>btShowLog</tabstop>
  <tabstop>ShowLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
